drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -154,8 +154,11 @@ class RasterizerSceneGLES3 : public RasterizerScene {
 			float fog_height_min;
 			float fog_height_max;
 			float fog_height_curve;
+
+			uint32_t eye;
+
 			// make sure this struct is padded to be a multiple of 16 bytes for webgl
-			float pad[2];
+			float pad[1];
```

Comment:
I'm actually not sure this ends up being a multiple of 16 even before this change. I might be counting wrong but possibly other variables have been added since this was originally set to 2